pkg/app/app.go

```diff
@@ -164,9 +165,36 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
+	} else {
+		ticker := time.NewTicker(interval)
+		ctx, cancelFunc := context.WithCancel(context.Background())
+		defer cancelFunc()
+		go func() {
+			for {
+				select {
+				case v := <-ticker.C:
+					a.log.Info("Broker configured to refresh specs every %v seconds", interval)
+					a.log.Info("Attempting bootstrap at %v", v.UTC())
+					if _, err := a.broker.Bootstrap(); err != nil {
```

Comment:
> Will we have a problem with constant bootstraps? 
>
Yes, Stdout gets overflowed with spec information if there are constant bootstraps. The default setting is 600s = 10m which is far out of the crazy zone (<10s).
> We would also run it when we called the endpoint which usually meant we probably didn't have any active provisions, etc. Will this auto bootstraps potentially cause problems?
>
 ```listen_and_serve``` will spawn go routines per request. The only thing we don't have async bootstraps, which doesn't make a whole lot of sense. So the bootstrap requests will be processed sequentially.  But, I don't think this will cause problems for provision because the last step of bootstrap does: ```_, err := d.kapi.Set(context.Background(), key, val /*opts*/, nil)```. This sets the key/value pairs based on all the spec data gathered. It doesn't delete anything or make the data unavailable, it only adds new data.